src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx

```diff
@@ -268,24 +268,18 @@ const currentConfig = Amplify.getConfig();
 ```
 
 <Callout warning={true}>
-  If your existing user pool client has a required attribute that is *not* set
-  to mutable, you may face login issues when using social sign-in. To resolve
-  this, you will need to
-  <ExternalLink href="https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html">
-    create a new user pool client
-  </ExternalLink>
-  and mark the required attribute as mutable.
+If your existing user pool client has a required attribute that is *not* set to mutable, you may face login issues when using social sign-in. To resolve this, you will need to <ExternalLink href="https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html"> create a new user pool client </ExternalLink> and mark the required attribute as mutable.
 </Callout>
 
 #### OAuth configuration parameters:
 
 These settings can be found in the Cognito User Pools console under the **App Integration** section
 
-- `domain`: This can be found in the **Domain name** subsection
-- `scope`: Remember to have the scope allowed on the Cognito App client; this can be found in the **App client settings** subsection
-- `redirectSignIn`: URL must be present on **Callback URL(s)**; check in the **App client settings** subsection
-- `redirectSignOut`: URL must be present on **Sign out URL(s)**; check in the **App client settings** subsection
-- `responseType`: Option must be enabled on the App client; look for **Allowed OAuth Flows** in the **App client settings** subsection. _Authorization code grant_ is for 'code' value and _Implicit grant_ is for 'token' value.
+| `domain` | This can be found in the **Domain name** subsection |
+| `scope` | Remember to have the scope allowed on the Cognito App client; this can be found in the **App client settings** subsection |
+| `redirectSignIn` | URL must be present on **Callback URL(s)**; check in the **App client settings** subsection |
+| `redirectSignOut`| URL must be present on **Sign out URL(s)**; check in the **App client settings** subsection |
+| `responseType` | Option must be enabled on the App client; look for **Allowed OAuth Flows** in the **App client settings** subsection. _Authorization code grant_ is for 'code' value and _Implicit grant_ is for 'token' value. |
```

Comment:
Hi @hibler13, I think this table. isn't working as expected. I think we might need to add some lines above it to help setup the table columns?

Something like this should work:

```suggestion
|  |  |
| --- | --- |
| `domain` | This can be found in the **Domain name** subsection |
| `scope` | Remember to have the scope allowed on the Cognito App client; this can be found in the **App client settings** subsection |
| `redirectSignIn` | URL must be present on **Callback URL(s)**; check in the **App client settings** subsection |
| `redirectSignOut`| URL must be present on **Sign out URL(s)**; check in the **App client settings** subsection |
| `responseType` | Option must be enabled on the App client; look for **Allowed OAuth Flows** in the **App client settings** subsection. _Authorization code grant_ is for 'code' value and _Implicit grant_ is for 'token' value. |
```